### PR TITLE
Add details and static index meta-db for JARVIS

### PR DESCRIPTION
**Pull request by ml-evs**
This PR adds a static index meta-db for JARVIS and links out to the JARVIS-DFT database, which is now publicly available via OPTIMADE.

@knc6: I took the description from your website at https://www.nist.gov/programs-projects/jarvis-dft, please let me know if you want it to say something different.

### CODE REVIEW

**Comment by ml-evs** [comment]
> Thank you, @ml-evs 
> Should we change `base_url`  in `links.json`,  https://github.com/Materials-Consortia/providers/blob/09e97f36bb3261b7fe432244d43b2736291790da/src/index-metadbs/jarvis/v1/links.json#L17 to https://jarvis.nist.gov/optimade/jarvisdft/v1 i.e. the link with '/v1' at the end ?

No, the base URL should omit the version, clients will automatically add it.

**Review by knc6 — APPROVED** [review]

**Comment by knc6** [comment]
> > Thank you, @ml-evs
> > Should we change `base_url`  in `links.json`,  https://github.com/Materials-Consortia/providers/blob/09e97f36bb3261b7fe432244d43b2736291790da/src/index-metadbs/jarvis/v1/links.json#L17
> >  to https://jarvis.nist.gov/optimade/jarvisdft/v1 i.e. the link with '/v1' at the end ?
> 
> No, the base URL should omit the version, clients will automatically add it.

Sounds good.

File: src/index-metadbs/jarvis/v1/info.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+   "meta" : {
+       "api_version" : "1.0.0",
```
Comment (CasperWA) [inline src/index-metadbs/jarvis/v1/info.json:3]: Shouldn't we update this for all now?
This is not a change that should be implemented in this PR though, but wanted to say it while I was here.

**Review by CasperWA — APPROVED** [review]
Looks good 👍

File: src/index-metadbs/jarvis/v1/info.json (hunk 1 above)
Comment (ml-evs) [reply src/index-metadbs/jarvis/v1/info.json:3]: Yeah, thought about this too. Could you raise an issue?

**Pull request merged**